var/spack/repos/builtin/packages/node-js/package.py

```diff
@@ -25,21 +25,15 @@ class NodeJs(Package):
     version("21.7.3", sha256="ce1f61347671ef219d9c2925313d629d3fef98fc8d7f5ef38dd4656f7d0f58e7")
     version("19.2.0", sha256="aac9d1a366fb57d68f4639f9204d1de5d6387656959a97ed929a5ba9e62c033a")
     version("17.9.1", sha256="1102f5e0aafaab8014d19c6c57142caf2ba3ef69d88d7a7f0f82798051796027")
-    version(
-        "15.3.0",
-        sha256="cadfa384a5f14591b84ce07a1afe529f28deb0d43366fb0ae4e78afba96bfaf2",
-        deprecated=True,
-    )
-    version(
-        "13.8.0",
-        sha256="815b5e1b18114f35da89e4d98febeaba97555d51ef593bd5175db2b05f2e8be6",
-        deprecated=True,
-    )
-    version(
-        "13.5.0",
-        sha256="4b8078d896a7550d7ed399c1b4ac9043e9f883be404d9b337185c8d8479f2db8",
-        deprecated=True,
-    )
+    version("15.3.0", sha256="cadfa384a5f14591b84ce07a1afe529f28deb0d43366fb0ae4e78afba96bfaf2")
```

Comment:
according to https://github.com/nodejs/node/blob/v15.3.0/configure#L24 this should not be deprecated (at least for what concerns python). I think it was deprecated according to the previous requirements we had about python.